features/step_definitions/journey_test_steps.rb

```diff
@@ -34,8 +34,12 @@
   when "Supplier"
     page.fill_in('email_address', :with => dm_supplier_user_email())
     page.fill_in('password', :with => dm_supplier_password())
-  when "CCS Sourcing"
-    page.fill_in('email_address', :with => dm_admin_ccs_sourcing_email())
+  when "CCS Sourcing", "CCS Category"
+    if user_type == "CCS Sourcing"
```

Comment:
I don't see the need for the nested "if" here?  It would be clearer just to have separate `case` switches for each user type I think.  Just because the three admin accounts all use the same password at the moment doesn't mean they always will, so it's probably better to set password separately for each account anyway..